resources/compass-runtime-agent/values.yaml

```diff
@@ -9,8 +9,8 @@ global:
       dir:
       version: "PR-6835"
     compassExternalSolutionTests:
-      dir: "develop/"
-      version: "79acee1e"
+      dir: "pr/"
```

Comment:
Please, bump external-solution-integration values in other charts that use that component (different scenarios) to make sure that the changes didn't affect them.